website/content/docs/commands/var/put.mdx

```diff
@@ -0,0 +1,87 @@
+---
+layout: docs
+page_title: "Command: var put"
+description: |-
+  The "var put" command writes a variable to the given path in Nomad.
+---
+
+# Command: var put
+
+The `var put` command creates or updates an existing variable.
+
+## Usage
+
+```plaintext
+nomad var put [options] <variable spec file reference> [<key>=<value>]...
+nomad var put [options] <path to store variable> [<variable spec file reference>] [<key>=<value>]...
+```
+
+Variable metadata and items can be supplied using a variable specification, by
+using command arguments, or by a combination of the two techniques. An entire
+variable specification can be provided to the command via standard input (stdin)
+by setting the first argument to "-" or from a file by using an @-prefixed path
+to a variable specification file. When providing variable data via stdin, you
+must provide the `-in` flag with the format of the specification, either "hcl"
+or "json"
+
+Items to be stored in the variable can be supplied using the specification, as a
+series of key-value pairs, or both. The value for a key-value pair can be a
+string, an @-prefixed file reference, or a '-' to get the value from stdin. Item
+values provided from file references or stdin are consumed as-is with no
+additional processing and do not require the input format to be specified.
+
+Values supplied as command line arguments supersede values provided in the any
```

Comment:
I feel like this sentence is missing some words or pivoted in it's implementation.  "In the any" doesn't make sense I don't think.

Reply:
Oops, right. This got copy-pasted from the CLI docs so I'll need to fix it there too 😁

Reply:
Fix the CLI here: https://github.com/hashicorp/nomad/pull/14552